src/components/ScheduleTable/TableWithCard/TableWithCard.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { makeStyles, Table, TableBody, TableCell, TableContainer, TableHead, TableRow, Paper } from '@material-ui/core';
+import CustomCard from '../CustomCard/CustomCard';
+
+const useStyles = makeStyles({
+  table: {
+    minWidth: 650,
+  },
+});
+
+const TableWithCard = ({ rows, columns, onClickHandler }) => {
+  const classes = useStyles();
+
+  return (
+    <TableContainer component={Paper}>
+      <Table className={classes.table} size='small' aria-label='a dense table'>
+        <TableHead>
+          <TableRow>
+            {columns.map((col) => {
+              return <TableCell key={col.field}>{col.headerName}</TableCell>;
+            })}
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {rows
+            ? rows.map((row, index) => (
+                <TableRow key={index}>
+                  {columns.map((col) => {
```

Comment:
Так можно избавиться от col. в коде ниже
```suggestion
                  {columns.map(({field}) => {
```

Reply:
исправлен